### PR TITLE
feat: add collection for assets that free from fees

### PR DESCRIPTION
**Pull request by shoom3301**
A new collection which contains a list of assets that should be free from volume fee in CoW Swap

image

image

image

### Add settings in CMS (staging and prod)
 - `One or two (comma separated) token ids. If one, then this token will be free from fees in any trade. If two, then only this specific pair will be free from fees.` - description to `tokenIds` field in view settings
 - `find` permission to `Public` role
 
 
image

### CODE REVIEW

File: src/api/tax-free-asset/content-types/tax-free-asset/schema.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "pluginOptions": {},
+  "attributes": {
+    "chainId": {
+      "type": "enumeration",
+      "enum": [
+        "MAINNET",
+        "GNOSIS_CHAIN",
+        "ARBITRUM_ONE",
+        "BASE",
+        "SEPOLIA"
```
Comment (alfetopito) [inline src/api/tax-free-asset/content-types/tax-free-asset/schema.json:22]: You should make it a FK to the Network model 
image

**Review by alfetopito — COMMENTED** [review]
One thing that's still not clear to me.
Here I only see individual assets, but no indication of pairs/groups of tokens.

The current implementation makes stable-stable trades free.

Will it change with this version, where if one of the tokens (buy or sell) includes any token from this collection, the trade will be fee free?

File: src/api/tax-free-asset/content-types/tax-free-asset/schema.json (hunk 1 above)
Comment (shoom3301) [reply src/api/tax-free-asset/content-types/tax-free-asset/schema.json:22]: Thank you! Fixed

**Comment by shoom3301** [comment]
>Here I only see individual assets, but no indication of pairs/groups of tokens.

@alfetopito `tokenIds` might contain two token address separated by comma, so you can specify a rule for a specific pair.
See "Add settings in CMS (staging and prod)" in the PR description.

image

File: src/api/tax-free-asset/content-types/tax-free-asset/schema.json (hunk 1 above)
Comment (alfetopito) [reply src/api/tax-free-asset/content-types/tax-free-asset/schema.json:22]: Have you pushed it already?

**Review by alfetopito — APPROVED** [review]

**Pull request merged**